sros2_cmake/README.md

```diff
@@ -1,19 +1,21 @@
 # Security Helper
-Add node authentication, cryptography, and access control security keys using a cmake macro.
-The macro will generate the secure root directory if it does not exists, then create authentication and cryptography keys in the secure root directory.
+Add authentication, cryptography, and access control security keys using a cmake macro.
+The macro will generate the secure root directory if it does not exists, then create authentication and cryptography keys.
 
 In package.xml add:  
 `<depend>sros2_cmake</depend>`  
 In CMakeLists add:  
 `find_package(sros2_cmake REQUIRED)`  
-`ros2_secure_node(NODES <node_name>)`  
+`ros2_security_generate_artifacts(SECURITY_CONTEXTS <context_name>)`  
 
 Macro definition:  
 ```
-    # ros2_secure_node(NODES <node_1> <node_2>...<node_n>)
+    # ros2_security_generate_artifacts(SECURITY_CONTEXTS <context_1> <context_2>...<context_n>)
 
-    # NODES (macro multi-arg) takes the node names for which keys will be generated
+    # SECURITY_CONTEXTS (macro multi-arg) takes the security contexts names for which keys will be generated
+    #   Each executable can be using a different (or the same), security contexts.
+    #   All nodes in the same process use the same security context.
     # SECURITY (cmake arg) if not define or OFF, will not generate key/keystores
     # ROS_SECURITY_ROOT_DIRECTORY (env variable) the location of the keystore
-    # POLICY_FILE (cmake arg) if defined, will compile policies by node name into the access private certificates (e.g POLICY_FILE=/etc/policies/<policy.xml>, Generate: <node_name> /etc/policies/<policy.xml>) **if defined, all nodes must have a policy defined for them**
+    # POLICY_FILE (cmake arg) if defined, will compile policies by security context name into the access private certificates (e.g POLICY_FILE=/etc/policies/<policy.xml>, Generate: <context_name> /etc/policies/<policy.xml>) **if defined, all contexts must have a policy defined for them**
```

Comment:
Unrelated to this PR but I can't comprehend this sentence

Reply:
Yeah, it's hard to read.
Proposed rephrasing:
```
if defined, will generate permissions based on the policies defined for each context in the file. All contexts must have a policy defined for them
```

Reply:
Yeah too many keywords, there is actually a single policy, that contains contexts, that contain profiles. A simpler version to consider:

```
if defined, will generate security artifacts for each context defined in the policy file.
```

The last bit about "must have" is actually not true since https://github.com/ros2/sros2/pull/108

Reply:
See https://github.com/ros2/sros2/pull/177/commits/070105b15778030a10ff45a98d02964689752822.